samples/wifi/throughput/README.rst

```diff
@@ -0,0 +1,308 @@
+.. _wifi_throughput_sample:
+
+Wi-Fi: Throughput
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Throughput sample demonstrates how to measure the throughputs for Nordic Semiconductor's Wi-Fi® chipsets.
```

Comment:
Let's call this "IP networking throughput", to make it more specific.
Suggestion
"The Throughput sample provides a framework for users to measure the achievable IP networking throughput of applications using Nordic Semiconductor's Wi-Fi® chipsets."